.github/workflows/test.yml

```diff
@@ -78,3 +78,14 @@ jobs:
       run: |
         ./verify.sh cluster
 
+  # test if kernel module can be loaded, this is very os and instance specific, ignore if it fails
+  test_kernel_module:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: ./
+      with:
+        kernel_module_names: rapl,intel_rapl_common
+    - name: verify
+      run: |
+        ./verify.sh modprobe rapl,intel_rapl_common
```

Comment:
Since both the two steps here need the kernel module list, why not we define the ENV of `kernel_module_names` in the whole yaml's `env` section similar as [here](https://github.com/sustainable-computing-io/kepler/blob/main/.github/workflows/integration_test.yml#L6-L9)? In this way, we do not need to add $2 parameter for `modprobe` branch.

Reply:
moved to `env`